src/components/Quote.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+export default function Quote() {
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [quote, setQuote] = useState('');
+
+  useEffect(() => {
+    const getData = async () => {
+      fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+        headers: {
+          'X-Api-Key': 'fDPmZpVzpJsZrqz3Waz2xQ==BEj9NXq27CsBpFNZ',
+        },
+      })
+        .then((resp) => resp.json())
+        .then((quote) => {
+          setLoading(false);
+          console.log(quote[0].quote);
```

Comment:
- [ ] Please do not commit **console.log** to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. 😅